cli/crates/cli/src/check.rs

```diff
@@ -16,6 +16,9 @@ pub(crate) async fn check(command: CheckCommand) -> Result<(), CliError> {
 
     let schema = match schema {
         Some(schema) => fs::read_to_string(schema).map_err(CliError::SchemaReadError)?,
+        None if atty::is(atty::Stream::Stdin) => {
```

Comment:
[`std::io::IsTerminal`](https://doc.rust-lang.org/std/io/trait.IsTerminal.html) was stabilised in 1.70 iirc - wonder if we could just start using that instead of the 3rd party crate.

Reply:
It seems to be working in my local testing, so let's do this.

Reply:
done